src/AutomatedCar/Models/WorldObject.cs

```diff
@@ -16,6 +19,19 @@ public WorldObject(int x, int y, string filename)
         }
 
         public int ZIndex { get; set; }
+        public int Angle
+        {
+            get => this._angle;
+        }
+
+        public int RotationCenterPointX 
+        {
+             get => this._rotationCenterPointX;
+        }
+        public int RotationCenterPointY
+        {
+             get => this._rotationCenterPointY;
+        }
```

Comment:
ezek nálunk is megvannak, így amelyikünk előbb merge-el a másiknak conflict-ot  csinál :smile: 
De nem nagy ügy, ha a ti PR-etek ha be lett mergel-ve akkor utána mi majd kezeljük szerintem.

Reply:
vagy azt is lehet h készítünk egy mini task-ot és csak ezt a közös részt bele tesszük egy PR-be és akkor mind ketten haladhatunk tovább

Reply:
Azt hiszem ez a PR van jobban készen, de még egyszer át akarom futni, valamin csapattárs review is kell még. @SzFMV2020-Osz/team-b4

Reply:
okés, legyen ez az első, utána mi szinkronizálunk ehhez